backend/requirements.txt

```
@@ -18,7 +18,7 @@ fastapi-pagination==0.9.1
 filelock==3.3.1
 h11==0.12.0
 httpcore==0.15.0
-httptools==0.2.0
+httptools==0.5.0
```

take out the versioning on this  completely. I think this still gives issue for some people.
i.e we should have
```suggestion
httptools
```